pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -114,25 +109,27 @@ public String getInstance(
   }
 
   private int getGrpcPort(InstanceConfig instanceConfig) {
-    int grpcPort;
-    try {
-      grpcPort = Integer.parseInt(instanceConfig.getRecord().getSimpleField(CommonConstants.Helix.Instance.GRPC_PORT_KEY));
-    } catch (Exception e) {
-      LOGGER.warn("Grpc port is not set for instance: {}", instanceConfig.getInstanceName(), e);
-      grpcPort = Instance.NOT_SET_GRPC_PORT_VALUE;
+    String grpcPortStr = instanceConfig.getRecord().getSimpleField(CommonConstants.Helix.Instance.GRPC_PORT_KEY);
```

Comment:
do we need to do any null check here?

Reply:
`instanceConfig` won't be `null` here, so just checking `grpcPortStr` should be enough